File: Linguagem de Programação/calculadora.c
#include <locale.h>
#include <stdio.h>
//Função da Soma
double soma(double n1,double n2){
    double Soma;
  	  Soma = n1 + n2;
	return Soma;
}

//Procedimento da subtração
double subtracao(double n1,double n2){

printf("Digite o valor do primeiro número: ");
		fflush(stdout);
		scanf("%lf", &n1);
	printf("Digite o valor do segundo número: ");
		fflush(stdout);
		scanf("%lf", &n2);

	printf("O resultado da subtração é %.2lf\n",(n1-n2));
}

//Função da Divisão
double divisao(double n1,double n2){
    double Div;
  	  Div = n1 / n2;
	return Div;
}

//Procedimento da multiplicação
double multiplicacao(double n1,double n2){

printf("Digite o valor do primeiro número: ");
		fflush(stdout);
		scanf("%lf", &n1);
	printf("Digite o valor do segundo número: ");
		fflush(stdout);
		scanf("%lf", &n2);

	printf("O resultado da subtração é %.2lf\n",(n1*n2));

}

int main(){
	double n1;
	double n2;
	int OperacaoEscolhida;
	char escolha;
	setlocale(LC_ALL, "Portuguese");

	//Menu
	do{
    printf("-------------------------\n");
    printf("|  SANDROLAXCALCULATORV2 |\n");
    printf("-------------------------\n");
    printf("|   1 - ADIÇÃO           |\n");
    printf("|   2 - SUBTRAÇÃO        |\n");
    printf("|   3 - DIVISÃO          |\n");
    printf("|   4 - MULTIPLICAÇÃO    |\n");
    printf("|   5 - SAIR             |\n");
    printf("-------------------------\n");
    scanf("%d",&OperacaoEscolhida);


 switch(OperacaoEscolhida){

	case 1:
	printf("Digite o valor do primeiro número: ");
		fflush(stdout);
		scanf("%lf", &n1);
	printf("Digite o valor do segundo número: ");
		fflush(stdout);
		scanf("%lf", &n2);

	printf("O resultado da soma é %.2lf\n",soma(n1,n2));
        break;

    	case 2:
	subtracao(n1,n2);
    	break;

    	case 3:
    	printf("Digite o valor do primeiro número: ");
		fflush(stdout);
		scanf("%lf", &n1);
	printf("Digite o valor do segundo número: ");
		fflush(stdout);
		scanf("%lf", &n2);

	printf("O resultado da divisão é %.2lf\n",divisao(n1,n2));
    	break;

    	case 4:
	multiplicacao(n1,n2);
    	break;

    case 5:
	printf("Obrigado por usar nossa aplicação!\n");
    break;

	default:
     printf("Digite um numero válido!\n");
    break;
 }
 printf("Deseja realizar uma nova operação?[S/N]");
 fflush(stdin);
 scanf("%c", &escolha);
 }while(escolha == 'S');

}

File: Estrutura de Dados/Aulas do Elegod/src/TrabalhoCalculadora/Calculadora.java
package TrabalhoCalculadora;

import java.util.Scanner;

public class Calculadora{
    public static void Calculadora(){
  
        System.out.println("Operadores:+,-,/,*");
        float valor1;
        float valor2;
        int OperadroEscolhido;
        String operadorMatematico;
        float resultado;
        Scanner valor = new Scanner(System.in);
        Scanner escolha = new Scanner(System.in);

       
			System.out.println("|--------------------------| ");
			System.out.println("|          MENU            |");
			System.out.println("|--------------------------|  ");
			System.out.println("| [1] Adição               |");
			System.out.println("| [2] Subtração            |");
			System.out.println("| [3] Multiplicação        |");
			System.out.println("| [4] Divisão  	           |");
			System.out.println("| [5] Sair   	           |");
			System.out.println("|--------------------------| ");
			do {
				System.out.print("Escolha uma das opções: ");
				OperadroEscolhido = escolha.nextInt();
				if(OperadroEscolhido == 1 || OperadroEscolhido == 2 || OperadroEscolhido ==3 || OperadroEscolhido == 4 || OperadroEscolhido == 5 ) {
					break;
				} else {
					System.out.print("Opção inválida! ");
				}
			}
			while(OperadroEscolhido != 1 || OperadroEscolhido != 2 || OperadroEscolhido !=3 || OperadroEscolhido != 4 || OperadroEscolhido != 5 );
		     if (OperadroEscolhido == 1) {
		    	 System.out.println("Informe primeiro valor:");
		         valor1 = valor.nextInt();
		         System.out.println("Informe segundo valor:");
		         valor2 = valor.nextInt();
		         resultado = valor1 + valor2;
		         System.out.println("O valor da soma é igual a:"+resultado);
			} 
		     else if (OperadroEscolhido == 2) {
				System.out.println("Informe primeiro valor:");
		        valor1 = valor.nextInt();
		        System.out.println("Informe segundo valor:");
		        valor2 = valor.nextInt();
		        resultado= valor1 - valor2;
		        System.out.println("O valor da subtração é igual a:"+resultado);
			  } 
		     else if (OperadroEscolhido == 3) {
			    System.out.println("Informe primeiro valor:");
			        valor1 = valor.nextInt();
			    System.out.println("Informe segundo valor:");
			        valor2 = valor.nextInt();
			    } 
		     else if (OperadroEscolhido == 4) {
			    System.out.println("Informe primeiro valor:");
			    valor1 = valor.nextInt();
			    System.out.println("Informe segundo valor:");
			    valor2 = valor.nextInt();
			    resultado = valor1 / valor2;
		        System.out.println("O valor da divisão é igual a:"+resultado);
			      } 
		     else if(OperadroEscolhido == 5) {
			    System.out.println("Obrigado por usar nossa aplicação! ");
			      }
		}
}
   
